src/components/containers/container.js

```diff
@@ -29,7 +29,7 @@ const Container = styled(Box)`
         max-width: 1600px;
     }
     @media ${device.tabletL} {
-        width: 90%;
+        width: ${(props) => props.width || '90%'};
```

Comment:
I'm afraid that this might affect all other pages globally unintentionally. @amina-deriv